packages/concerto-core/test/models/test.js

```diff
@@ -208,10 +208,6 @@ describe('Test Model', function(){
             cObject.model = { 'foo' : 'bar' };
             ( function() {serializer.toJSON(cObject);}).should.throw('Model violation in the "org.acme.Vehicle#YVBVLSFXXAL342374" instance. The field "model" has a value of "{"foo":"bar"}" (type of value: "object"). Expected type of value: "String".');
 
-            // set model to null
```

Comment:
Is this redundant now?

Reply:
Yes, the model contains a default modifier, which is now being used properly.
```
o String model default="F150"
```